av.h

```diff
@@ -98,54 +98,72 @@ If all you need is to look up an array element, then prefer C<av_fetch>.
 
 /*
 =for apidoc newAV
+=for apidoc_item newAV_alloc_x
+=for apidoc_item newAV_alloc_xz
 
-Creates a new AV.  The reference count is set to 1.
+These all create a new AV, setting the reference count to 1. They differ
+in when and how an accompanying SV* array is allocated and initialized.
```

Comment:
how about "They differ in allocating and populating the array of SV*s that always accompanies a non-empty AV."
or something similar

Reply:
I've changed it to something similar.